conf/nginx.conf

```diff
@@ -123,6 +123,9 @@ location ^~ __PATH__/ {
     if ($request_method ~ ^DELETE$) {
         rewrite ^ __PATH__/index.php$request_uri last;
     }
+    if ($request_method ~ ^PATCH$) {
```

Comment:
```
  location ~ / {
    if ($request_method ~ ^(PUT|DELETE|PATCH)$) {
        rewrite ^ __PATH__/index.php$request_uri last;
    }
    try_files $uri / __PATH__/index.php$request_uri;
  }
```

Reply:
Good call. Though I'm mostly wondering if there are any security considerations to have, and if other keywords need to be supported.
I will make another commit with the suggested change.